sdk/ml/azure-ai-ml/azure/ai/ml/entities/_datastore/one_lake.py

```diff
@@ -4,23 +4,40 @@
 
 # pylint: disable=protected-access,no-member
 
+from abc import ABC
 from pathlib import Path
 from typing import Dict, Optional, Union
 
 from azure.ai.ml._restclient.v2023_04_01_preview.models import (
     OneLakeDatastore as RestOneLakeDatastore,
     Datastore as DatastoreData,
     DatastoreType,
-    OneLakeArtifact,
+    LakeHouseArtifact as RestLakeHouseArtifact,
 )
 from azure.ai.ml._schema._datastore.one_lake import OneLakeSchema
 from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY, TYPE
 from azure.ai.ml.entities._credentials import NoneCredentialConfiguration, ServicePrincipalConfiguration
 from azure.ai.ml.entities._datastore.datastore import Datastore
 from azure.ai.ml.entities._datastore.utils import from_rest_datastore_credentials
+from azure.ai.ml.entities._mixins import DictMixin, RestTranslatableMixin
 from azure.ai.ml.entities._util import load_from_dict
 
 
+class OneLakeArtifact(RestTranslatableMixin, DictMixin, ABC):
```

Comment:
Hi Young, can you add docstrings for these new classes, describing the classes, their params and types similar to the one for OneLakeDatastore below?